frontend/src/components/signup/AddInfo.tsx

```diff
@@ -82,17 +88,41 @@ export default function AddInfo({
   };
 
   return (
-    <>
-      <Form
-        fields={fields}
-        messages={messages}
-        formAction={formAction}
-        onSubmit={(event, values) => handleSubmit(event, values)}
-        errorMessage={errorMessage}
-        onGoBack={handleGoBack}
-        /*TODO(undefined) fieldClassName={classes.fieldClassName} */
-        autocomplete="off"
-      />
-    </>
+    <Card>
+      {/* TODO: maybe use card Header instead (?)
+      see https://mui.com/material-ui/react-card/ for other usefull card components */}
+      <Box
+        sx={{
```

Comment:
Is it possible to use the `className` prop instead of inline style here?(for Box component)

Reply:
yes, sure. Good catch.